app/res/values-v11/themes.xml

```diff
@@ -81,6 +81,8 @@
         <item name="android:paddingLeft">@dimen/title_paddingLeft</item>
 	</style>
     
-    
+    <style name="Theme.Dialog" parent="@android:style/Theme.Holo.Light.Dialog">

```

Comment:
Woah. This is a change to ~all of our dialogs that removes their title bar, right? Do we want htis?